packages/style/scss/colors.scss

```diff
@@ -108,7 +108,7 @@ $secondary: (
 );
 
 $semantic: (
-    'success-10': hsla(152, 87%, 91%, 1),
+    'success-00': hsla(152, 87%, 91%, 1),
```

Comment:
Is that used in the admin? It would be considered a breaking change?

Reply:
since 3 days ago :sob: 
![image](https://user-images.githubusercontent.com/63734941/163586053-aa11f938-394b-41a6-ae5d-7fe14f343476.png)

I can revert this actually it was just to have the same name than in Figma :smiling_face_with_tear: